app/models/katello/candlepin/repository_mapper.rb

```diff
@@ -1,40 +1,5 @@
 module Katello
-  class Candlepin::Content
```

Comment:
Oops, I definitely intended to remove these methods in my recent PR. Good cleanup.

Reply:
no worries, these prs are exposing some of the more ugly (and older) sides of katello, so its good to clean them up.  Your recent PR did plenty of that!